packages/client/src/pages/PortalSettings/categories/data-management/backup/common-container/DirectThirdPartyConnection.js

```diff
@@ -364,6 +364,7 @@ const DirectThirdPartyConnection = (props) => {
 
       {!connectedThirdPartyAccount?.id || !isTheSameThirdPartyAccount ? (
         <Button
+          id="connect"
```

Comment:
Too simple id value (potential duplicates)

Reply:
Fix to "connect-button"